modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -7,21 +7,26 @@ import scala.scalajs.js
 import scala.scalajs.js.|
 
 import cats.syntax.all._
-import japgolly.scalajs.react.Callback
+import japgolly.scalajs.react._
 import japgolly.scalajs.react.MonocleReact._
-import japgolly.scalajs.react.ReactEventFromInput
-import japgolly.scalajs.react.ScalaComponent
 import japgolly.scalajs.react.component.builder.Lifecycle.RenderScope
 import japgolly.scalajs.react.raw.JsNumber
 import japgolly.scalajs.react.vdom.html_<^._
 import monocle.macros.Lenses
+import org.scalajs.dom.html
 import react.common._
 import react.semanticui._
 import react.semanticui.collections.form.FormInput
 import react.semanticui.elements.icon.Icon
 import react.semanticui.elements.input._
 import react.semanticui.elements.label._
 
+trait UpdateValue extends Product with Serializable
+object UpdateValue {
+  case object OnChange extends UpdateValue
+  case object OnBlur   extends UpdateValue
+}
+
```

Comment:
Will move this elsewhere if we go this route. We can use the same thing for validation.